plugins/sniffer/lib/sniffer_plugin.rb

```diff
@@ -0,0 +1,29 @@
+class SnifferPlugin < Noosfero::Plugin
+
+  def self.plugin_name
+    "Opportunity Sniffer"
+  end
+
+  def self.plugin_description
+    _("Sniffs product suppliers and consumers near to your enterprise.")
+  end
+
+  def stylesheet?
+    true
+  end
+
+  def js_files
+    ['underscore-min.js', 'sniffer.js'].map{ |j| "javascripts/#{j}" }
```

Comment:
@larissa could you please load these locally? we already have many global javascripts